app/components/preprint-form-section.js

```diff
@@ -74,7 +74,7 @@ export default CpPanelComponent.extend(Analytics, {
                     });
                 this._super(...arguments);
             } else {
-                this.sendAction('errorAction', this.get('denyOpenMessage')); /* eslint-disable-line ember/closure-actions */
+                this.get('errorAction')(this.get('denyOpenMessage'));
```

Comment:
Come to think of it, I didn't test that this one worked. Will do that now.

Reply:
Ahh yup, the upload section is openable before selecting a preprint server. No error message 😭 

I think I see a potential fix, if that doesn't work I'll revert this one as well 😿